### ISSUE TITLE
[PAYMENT 12/25 regression] [$250] Feature Request: Implement to use a 👍icon next to approved report preview

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#49847]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** n/a
**Reproducible in staging?:** n/a
**Reproducible in production?:** n/a
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @youssef-lr 
**Slack conversation:**  https://expensify.slack.com/archives/C049HHMV9SM/p1727409632797909

## Action Performed:
1. Have an approved expense report
2. Open the report

## Expected Result:
Should have some indication with icon for the report is approved
## Actual Result:
 Implement to use a  :+1::skin-tone-3: icon next to approved report preview, similar to how we display a checkmark next to paid reports.

## Workaround:
Unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos
![Screenshot 2024-07-12 at 18 10 44](https://github.com/user-attachments/assets/400c2196-0a4a-4a2f-90b7-c1e23808a9f4)

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021839674890714199459</li>
        <li>Upwork Job ID: 1839674890714199459</li>
        <li>Last Price Increase: 2024-09-27</li>
<li>Automatic offers: </li>
<ul>
<li>situchan | Reviewer | 104253538</li>
<li>Krishna2323 | Contributor | 104253539</li>
<li>DylanDylann | Contributor | 104775457</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>

**Comment by dannymcclain** [comment]
Adding some design specs here for easier implementation:

- The icon we want to use is our `thumbs up` icon—it can be found here: `App/assets/images/thumbs-up.svg`
- The layout should copy what we already do for paid reports, just instead of a green checkmark icon, we want to use an `icons` colored thumbs up icon

Mockup:
![CleanShot 2024-09-27 at 08 44 10@2x](https://github.com/user-attachments/assets/99550c4d-a709-4aac-9957-aa5d8341179e)

cc @dubielzyk-expensify

**Comment by JmillsExpensify** [comment]
Thanks @dannymcclain! That looks solid.

**Comment by dannymcclain** [comment]
Yeah I think we should animate the 👍 if possible! The transition to the approved state I think will typically be pretty subtle, so a little animation would be good IMO.

cc @dubielzyk-expensify in case he disagrees.

**Comment by dubielzyk-expensify** [comment]
100% agree. That'd be great!

**Comment by situchan** [comment]
@Nodebrute can you please elaborate the meaningful difference from previous proposal?
And now that we want to animate, please remove deprecated proposal which is confusing.

**Comment by situchan** [comment]
@Krishna2323 can you please share test branch and demo video?

**Comment by situchan** [comment]
> I believe the selected proposal should be more thoroughly researched.

Thanks. The proposal is **not PR** so no need to have detailed code changes unless C+ asks for it.

**Comment by situchan** [comment]
@Krishna2323's [proposal](https://github.com/Expensify/App/issues/49847#issuecomment-2379422035) looks good to me.
🎀👀🎀 C+ reviewed

**Comment by situchan** [comment]
> > Please have the policy's billing owner ([dev.krishna.react@gmail.com](mailto:dev.krishna.react@gmail.com)) visit the settings page and update their billing information.
> 
> @situchan, how can I fix this BE error? 

can you share screenshot?

**Comment by situchan** [comment]
@Krishna2323 you posted upwork settings page. Is it related to this GH?

**Comment by situchan** [comment]
@Krishna2323 that's because subscription is expired. please test with new accounts.

**Comment by slafortune** [comment]
@Krishna2323 @situchan can you update this issue?

**Comment by DylanDylann** [comment]
@youssef-lr @slafortune I can help to complete the PR

**Comment by slafortune** [comment]
Thanks for volunteering @DylanDylann

**Comment by DylanDylann** [comment]
@Krishna2323 Let's prioritize this PR before creating new proposals

**Comment by slafortune** [comment]
not overdue, waiting on PR

**Comment by slafortune** [comment]
deployed to production today

**Comment by slafortune** [comment]
Paid contracts - less 50% due to regression.

**Comment by DylanDylann** [comment]
@slafortune It seems we don't have regression on this issue. The second PR is the same as the previous one. And we haven't proved that the regression is caused by the original PR

cc @youssef-lr @Krishna2323

**Comment by slafortune** [comment]
Offers sent @Krishna2323 @DylanDylann 
https://www.upwork.com/nx/wm/offer/105681323
https://www.upwork.com/nx/wm/offer/105681354
